### PR TITLE
fix: Enabled readme generation script to handle V2 Bicep formatter

### LINKED ISSUE #751: [Question/Feedback]: Set-AVMModule hangs
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Description

Recently, running Set-AVMModule on my local machine hangs when processing files. I don't see any recent commits that would have changed much about this function and I'm not sure how to troublshoot.

`PS /Users/matthewschmitt/GIT/bicep-registry-modules> Set-AVMModule -ModuleFolderPath avm/ptn/network/hub-networking -ThrottleLimit 10
Building [avm-network/hub-networking/main.bicep]
/Users/matthewschmitt/GIT/bicep-registry-modules/avm/ptn/network/hub-networking/main.bicep(6,7) : Warning no-unused-params: Parameter "name" is declared but never used. [https://aka.ms/bicep/linter/no-unused-params]
Generating readme for [avm-network/hub-networking/main.bicep]
WARNING: Resource type [hubnetworking] does not exist in the API / is custom. Falling back to it as default.
WARNING: No resource type like [Microsoft.Network/hubnetworking] found in template. Falling back to it as identifier.
Processed [0/1] files [0% complete                                                                              ]
`

and it'll just sit there.

### LINKED ISSUE #1315: [AVM Module Issue]: new formatting of Bicep breaks ReadMe Generation
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Issue Type?

Bug

### Module Name

Other, as defined below...

### (Optional) Module Name if not listed above

_No response_

### (Optional) Module Version

_No response_

### Description

Due to the new formatting in Bicep, the logic in the readme generator is broken. 

### (Optional) Correlation Id

_No response_

### PR DESCRIPTION
**Pull request by AlexanderSehr**
## Description

- Enabled readme generation script to handle V2 Bicep formatter (introduced via 0.26.54)

## Pipeline Reference

| Pipeline |
| -------- |
|          |

### CODE REVIEW

**Review by jeetgarg — DISMISSED** [review]

**Review by sebassem — DISMISSED** [review]

**Review by fbinotto — APPROVED** [review]
LGTM

**Review by ReneHezser — APPROVED** [review]

**Review by ChrisSidebotham — APPROVED** [review]
lgtm

**Comment by ChrisSidebotham** [comment]
Pending @eriqua Review

File: avm/res/analysis-services/server/README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ This instance deploys the module with the minimum set of required parameters.
 
 ```bicep
 module server 'br/public:avm/res/analysis-services/server:<version>' = {
-  name: '${uniqueString(deployment().name, resourceLocation)}-test-assmin'
```
Comment (ChrisSidebotham) [inline avm/res/analysis-services/server/README.md:51]: Adding blocker for @eriqua review

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**